src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
```

Comment:
It is better to make the _**Random**_ as **final static** in order not to create it anew each time the method _**getRandomColor**_ is called